js/models/contact_model.js

```diff
@@ -291,6 +302,29 @@ angular.module('contactsApp')
 					return false;
 				});
 				return matchingProps.length > 0;
+			},
+
+			validate: function(prop, property) {
+				switch(prop) {
```

Comment:
since there's only one condition here, it would be easier to simply change this to an if statement